lib/assertions.ts

```diff
@@ -1063,6 +1065,12 @@ export default class DOMAssertions {
    * @private
    */
   private get targetDescription() {
-    return elementToString(this.target);
+    if (typeof this.target === 'string') {
+      return this.target;
+    } else if (this.target instanceof Element) {
+      return elementToString(this.target);
+    } else {
+      return '<not found>';
+    }
```

Comment:
why did you adjust `targetDescription()` instead of adding this to `elementToString()`?

Reply:
Good question. I can't remember my thought process. It _does_ feel more appropriate to add to `elementToString()`. Let me look at it again and see what I was thinking.

Reply:
Here comes the long winded reply:

This was added to address a particular case that affects both `is-visible` _and_ `exists`. It was explicitly related to fixing `is-visible`'s tests when an element is not found via `document.querySelector`, and therefore returns `null`. This is not present in the case where a `string` selector is passed. Specifically, this test:

https://github.com/simplabs/qunit-dom/blob/29cade3b969949f1d535774b79225653263c6bc4/lib/__tests__/is-visible.ts#L58-L60

that you called out in a comment above _does_ have value. That's because we don't test for a case where you pass an expected element as the target, but that element isn't found in the DOM. 

The `exists` tests only cover the following case:

https://github.com/simplabs/qunit-dom/blob/29cade3b969949f1d535774b79225653263c6bc4/lib/__tests__/exists.ts#L31

so the `is-visible` tests are slightly more expansive.

The reason this affects `exists` and `is-visible` is that both utilize the following pattern in their assertions:

https://github.com/simplabs/qunit-dom/blob/29cade3b969949f1d535774b79225653263c6bc4/lib/assertions/exists.ts#L7

where they try to find all elements that match the target in order to get a count. When the target is `null`, this returns an empty array via `this.findElements`:

https://github.com/simplabs/qunit-dom/blob/29cade3b969949f1d535774b79225653263c6bc4/lib/assertions.ts#L1053-L1054

So, when the code continues within the respective assertions for `exists` and `is-visible`, we try to utilize `this.targetDescription` to format the description. The `targetDescription` getter expects either a `string` or an `Element`, but in the above case the target is neither; it's `null`. Since it's not an `Element`, it made sense to add this as a fallback here, inside `targetDescription`, so that we can cover any case where `this.targetDescription` is used on a target that's neither a `string` or `Element`. I feel like this is a good fallback. 

The only thing I'd add to this PR at this point is a test to ensure that `exists` does the correct thing when `null` is passed.